IBPSA/Resources/ReferenceResults/Dymola/IBPSA_Utilities_Psychrometrics_Examples_Density_pTX.txt

```diff
@@ -1,9 +1,7 @@
-last-generated=2015-11-05
-statistics-simulation=
-{
-  "numerical Jacobians": "0", 
-  "nonlinear": " ", 
-  "linear": " "
-}
-den.d=[1.16439318656921386719, 1.16419506072554401754, 1.16399693488190081325, 1.16379892824753383707, 1.16360104082245641166, 1.16340315343423839067, 1.16320538518159111163, 1.16300761700266264143, 1.16280996795941149458, 1.16261243827246096849, 1.16241490843851735804, 1.16221737860457352554, 1.16201996797991924382, 1.16182267656455473492, 1.1616253852961833104, 1.16142821323710143666, 1.16123116009332272291, 1.16103410724342381855, 1.160837054393738077, 1.16064012045939080053, 1.16044330602785783491, 1.16024649130322665691, 1.16004979608098324206, 1.15985310085873982722, 1.15965652455268775078, 1.1594600677490234375, 1.15926361065226091185, 1.15906715414169481093, 1.15887081625422183606, 1.15867459757653556984, 1.15847837948334086988, 1.15828228001501543076, 1.15808618054711609524, 1.1578902008720741712, 1.1576943398223276116, 1.15749847877258127404, 1.15730273751569190388, 1.15710699567580355129, 1.15691125383591519871, 1.15671575041460594768, 1.15652012836665085693, 1.15632474415463115669, 1.1561293599426112344, 1.15593397631323546726, 1.1557387113106472043, 1.15554356551706449707, 1.15534842030577045158, 1.15515339372211656155, 1.15495836713860500211, 1.1547634603451792934, 1.15456855297088623047, 1.15533077789585747475, 1.15609395649741863998, 1.15685820798315419644, 1.15762340858273593547, 1.15838968662440455759, 1.15915703755194710034, 1.15992534215609088832, 1.1606946004368192682, 1.16146493160186414784, 1.16223633105309698266, 1.16300868877450103156, 1.16378200017277944056, 1.16455650366537444285, 1.16533196083455936609, 1.16610849088962420517, 1.1668859699830766985, 1.16766452659293729077, 1.16844415608910412452, 1.16922485847030332984, 1.17000651452850790157, 1.17078924347259771821, 1.17157304062713296666, 1.17235791533360300676, 1.1731438629276580432, 1.17393076419830322266, 1.1747187383542596617, 1.17550778539609557249, 1.17629790532381117707, 1.17708909341720890929, 1.17788135910901026193, 1.1786746976866855352, 1.17946910915025182653, 1.18026459349869727866, 1.18106115073344875022, 1.18185865688778712901, 1.18265735988804521206, 1.18345713577276190343, 1.18425798454364250567, 1.18505990620068679675, 1.18586301995147946897, 1.18666708258363251538, 1.1874722228916367861, 1.18827855529324732942, 1.18908596058088522618, 1.18989443875481759605, 1.19070398981463543286, 1.19151472813261372075, 1.19232654416580663437, 1.19313943308487924178, 1.19395339488983154297]
+last-generated=2015-05-08
+statistics-simulation={'numerical Jacobians': '0', 'nonlinear': ' ', 'linear': ' '}
```

Comment:
vice versa: multi-line formatting replaced by single-line formatting

Reply:
The singe line vs multi-line formatting is probably because of an old version of BuildingsPy. One was generated in 2015 and the other in 2018.